apps/namada-interface/src/App/Staking/ValidatorDetails/ValidatorDetails.tsx

```diff
@@ -94,20 +98,98 @@ const validatorToDataRows = (
   ];
 };
 
+type StakingViewProps = {
+  onRequestClose: (modalOnRequestCloseType: ModalOnRequestCloseType) => void;
+};
+
+const StakingView = (props: StakingViewProps): JSX.Element => {
+  const { onRequestClose } = props;
+  return (
+    <>
+      <Button
+        variant={ButtonVariant.Contained}
+        onClick={() => onRequestClose(ModalOnRequestCloseType.Confirm)}
+      >
+        Confirm
+      </Button>
+      <Button
+        variant={ButtonVariant.Contained}
+        onClick={() => onRequestClose(ModalOnRequestCloseType.Cancel)}
+        style={{ backgroundColor: "lightgrey", color: "black" }}
+      >
+        Cancel
+      </Button>
+    </>
+  );
+};
+
+enum ModalState {
+  None,
+  Stake,
+  Unstake,
+}
+
+enum ModalOnRequestCloseType {
+  Confirm,
+  Cancel,
+}
+
 export const ValidatorDetails = (props: Props): JSX.Element => {
   const { validator, stakingPositionsWithSelectedValidator = [] } = props;
   const validatorDetailsData = validatorToDataRows(validator);
 
+  const [modalState, setModalState] = useState(ModalState.None);
+
+  const onRequestCloseStakingModal = (
+    modalOnRequestCloseType: ModalOnRequestCloseType
+  ): void => {
+    switch (modalOnRequestCloseType) {
+      case ModalOnRequestCloseType.Confirm: {
+        console.log(`called action ${modalOnRequestCloseType}`);
```

Comment:
Is it intentional to keep these `console.log` statements in for now?

Reply:
I left intentionally as this was just a temp state. But I removed now 👍